modules/microshift-lvms-configuring.adoc

```diff
@@ -32,28 +32,25 @@ device-classes: <2>
     lvcreate-options: <9>
       - --type=raid1
 ----
-<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`. 
-<2> `map[string]DeviceClass`. The `device-class` settings. 
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`.
+<2> `map[string]DeviceClass`. The `device-class` settings.
 <3> String. The name of the `device-class`.
-<4> String. The group where the `device-class` creates the logical volumes. 
-<5> unit64. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `10`. 
-<6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`. 
+<4> String. The group where the `device-class` creates the logical volumes.
+<5> unit64. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `10`.
+<6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`.
 <7> unit. The number of stripes in the logical volume.
-<8> String. The amount of data that is written to one device before moving to the next device. 
-<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
-+
-[WARNING]
+<8> String. The amount of data that is written to one device before moving to the next device.
+<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`].
+
+[IMPORTANT]
```

Comment:
I'm not sure I fully understand the consequence explained in the admonition, but it seems to imply that data loss could possibly occur. If so, I think WARNING is correct: https://redhat-documentation.github.io/supplementary-style-guide/#conversational-style